src/services/ogmios/constants.ts

```diff
@@ -33,7 +33,10 @@ export const handleEraBoundaries: EraBoundaries = {
 export const POLICY_IDS: PolicyIds = {
     mainnet: ['f0ff48bbb7bbe9d59a40f1ce90e9e9d0ff5002ec48f232b49ca0fb9a'],
     testnet: ['8d18d786e92776c824607fd8e193ec535c79dc61ea2405ddf3b09fe3'],
-    preprod: ['f0ff48bbb7bbe9d59a40f1ce90e9e9d0ff5002ec48f232b49ca0fb9a'],
+    preprod: [
+        // 'f0ff48bbb7bbe9d59a40f1ce90e9e9d0ff5002ec48f232b49ca0fb9a',
```

Comment:
I think we are going back to the original policy id yeah?

Reply:
ohh good catch. I added this for testing. I'll change it back